Src/Newtonsoft.Json/JsonTextWriter.cs

```diff
@@ -317,27 +319,28 @@ private void UpdateCharEscapeFlags()
         /// </summary>
         protected override void WriteIndent()
         {
-            _writer.WriteLine();
-
             // levels of indentation multiplied by the indent count
             int currentIndentCount = Top * _indentation;
 
-            if (currentIndentCount > 0)
-            {
-                if (_indentChars == null)
-                {
-                    _indentChars = new string(_indentChar, 10).ToCharArray();
-                }
+            int newLineLen = SetIndentChars();
 
-                while (currentIndentCount > 0)
-                {
-                    int writeCount = Math.Min(currentIndentCount, 10);
+            _writer.Write(_indentChars, 0, newLineLen + Math.Min(currentIndentCount, 12));
```

Comment:
Is there a reason you've increased the indentChars from length 10 to 12? I pick that number randomly when I wrote this so I'm just curious.